docs/style-guide/general/language.md

```diff
@@ -173,6 +173,7 @@ The following list contains common words and phrases that you should avoid in ed
 | blast radius | affected scope | metaphor, jargon |
 | ergo | therefore, as a result, so | Latin word meaning "therefore" |
 | etc. | " . . . and other {entities matching the description}." | Latin abbreviation of "et cetera", which means "and the rest". |
+| e.g. or i.e. | "For example", "such as", "that is" | Abbreviation of Latin phrases used to give examples or restate what was previously said. |
```

Comment:
```suggestion
| e.g. or i.e. | "For example", "such as", | Abbreviation of Latin phrases used to give examples or restate what was previously said. |
```
I think we should avoid phrases (ahem), such as "that is" and "in other words". These phrases mean that you did not clearly state something the first time.